src/autocomplete.js

```diff
@@ -508,13 +508,14 @@ export default class Autocomplete extends Component {
     }
 
     const computedMenuAttributes = {
-      // Copy the attributes passed as props
-      ...menuAttributes,
       // And add the values computed for the autocomplete
+      ...menuAttributes,
       id: `${id}__listbox`,
       role: 'listbox',
+      'aria-labelledby': id,
       className: menuClassList.join(' '),
       onMouseLeave: this.handleListMouseLeave
+      // Copy the attributes passed as props
```

Comment:
**nitpick** I'm not sure this comment ended up in the right line, it should be before `...menuAttributes`, I think.

Reply:
You are correct. This is a carryover from when I was messing with that object and thought it might be better for `...menuAttributes` to be declared after the other attributes as an override, which we don't wanna do. Was a symptom of me initially misunderstanding how the computed attributes worked. Will fix!